docs/windows-architecture.md

```diff
@@ -0,0 +1,231 @@
+│  • System tray icon for user interaction                            │
+│  • NOT managed by service - completely independent                  │
+│  • Talks to service via HTTP APIs                                   │
+└─────────────────────────────────────────────────────────────────────┘
+```
+
+## Components
+
+### pinsharesvc.exe (Windows Service Wrapper)
+
+The main Windows service that orchestrates all PinShare components.
+
+**Responsibilities:**
+- Registers as a Windows Service ("PinShareService")
+- Starts and monitors IPFS daemon
+- Starts and monitors PinShare backend
+- Runs embedded UI server (serves React UI)
+- Health checking with automatic restart on failure
+- Graceful shutdown of all components
+
+**Ports:**
+- 8888: UI Server (serves React frontend, proxies API requests)
+
+**Source:** `cmd/pinsharesvc/`
+
+### pinshare.exe (Main Daemon)
+
+The core PinShare application with libp2p networking.
+
+**Responsibilities:**
+- libp2p host for P2P communication
+- PubSub for metadata synchronization
+- File watcher for upload folder
+- REST API for external integrations
+- Connects to IPFS daemon for storage
+
+**Ports:**
+- 9090: REST API
+- 50001: libp2p P2P port
+
+**Source:** `internal/` (main application code)
+
+### ipfs.exe (IPFS Kubo Daemon)
+
+Standard IPFS daemon for content-addressed storage.
+
+**Ports:**
+- 5001: IPFS API
+- 4001: IPFS Swarm (P2P)
+- 8080: IPFS Gateway
+
+**Source:** Downloaded from https://dist.ipfs.tech/kubo/
+
+### pinshare-tray.exe (System Tray Application)
+
+User-facing system tray application for easy interaction.
+
+**Responsibilities:**
+- System tray icon with context menu
+- Open web UI in browser
+- Start/Stop/Restart service
+- Show service status
+
+**Note:** This runs independently of the service, launched via Windows Startup folder.
+
+**Source:** `cmd/pinshare-tray/`
+
+## Data Flow
+
+```
+User clicks tray icon
+        │
+        ▼
+pinshare-tray.exe ──HTTP──► pinsharesvc.exe (port 8888)
+                                   │
+                                   ├──proxy──► pinshare.exe API (port 9090)
+                                   │                  │
+                                   │                  ▼
+                                   │           libp2p network
+                                   │                  │
+                                   └──────────► ipfs.exe (port 5001)
+                                                      │
+                                                      ▼
+                                               IPFS network
+```
+
+## Installed Files
+
+```
+C:\Program Files\PinShare\
+├── pinsharesvc.exe    # Windows service wrapper
+├── pinshare.exe       # Main daemon (managed by service)
+├── pinshare-tray.exe  # User tray app (independent)
+├── ipfs.exe           # IPFS daemon (managed by service)
+└── ui\                # React web UI (served by service)
+    ├── index.html
+    ├── assets\
+    └── ...
```

Comment:
Let's remove the UI directory for the time being. Please update this ASCII section accordingly.

Reply:
Updated the installed files section - replaced ui/ directory with resources/ containing just the tray icon.